components/DatePicker.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChakraProps, Input } from '@chakra-ui/react'; // Chakra UI
+import ReactDayPickerInput from 'react-day-picker/DayPickerInput'; // React Day Picker Input
+import 'react-day-picker/lib/style.css'; // React Day Picker default styling
+
+// Date Picker props
+type Props = {
+  readonly value: Date; // Date value
+  readonly onDateChange: (day: Date) => void; // Callback on date change
+} & Pick<ChakraProps, 'width'>;
+
+/**
+ * Custom date picker component wrapping React Day Picker
+ * @param date The currently selected date
+ * @param onDateChange Callback on date change
+ * @returns
+ */
+export default function DatePicker(props: Props) {
+  const { width, value, onDateChange } = props;
+
+  return (
+    <>
+      <ReactDayPickerInput
+        component={(props: any) => <Input width={width} {...props} />}
```

Comment:
just curious why did you choose to render it as an Input component?

Reply:
The default `DayPickerInput` component provided by React Day Picker renders its own input field with its own styling. Rather than manually styling that all over to maintain consistency across our app, I just used the Chakra UI `Input` component.